src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./styles/index.scss"
```

Comment:
Stylesheets should be linked with a .css extension, not .scss. The browser cannot understand SCSS files directly; they need to be compiled to CSS first.